src/components/GardenCard/index.tsx

```diff
@@ -0,0 +1,91 @@
+import { Box, Card, CardBody, Image } from '@chakra-ui/react';
+import { ReactNode, useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import { LikeDoneIcon, LikeIcon } from '@/assets/icons';
+
+export interface GardenCardProps {
+  type: PageType;
+  src: string;
+  children: ReactNode;
+  navUrl: string;
+
+  // 좋아요 API 통신시 수정 예정입니다.
+  clickLike: () => void;
+}
+
+type PageType = 'mainPage' | 'cropsPage';
+
+const GardenCard = ({ type, src, children, navUrl }: GardenCardProps) => {
```

Comment:
꿀팁인데 children이 있는 컴포넌트 props 타입은
PropsWithChildren<타입> 으로 적용해주면 더 보기좋게 작성할 수 있어요!

```suggestion
export interface GardenCardProps {
  type: PageType;
  src: string;
  navUrl: string;
  clickLike: () => void;
}

type PageType = 'mainPage' | 'cropsPage';

const GardenCard = ({ type, src, children, navUrl }: PropsWithChildren<GardenCardProps>) => {
```

Reply:
찾아보니 PropsWithChildren 사용하면 children 타입을 ReactNode로 지정해주지 않아도 되네요!! 

감사합니다 이거 맞춰서 수정하도록 하겠습니다~!